server/controllers/index.js

```diff
@@ -0,0 +1,224 @@
+//module to contain the controller functions for postgres and cassandra
```

Comment:
One refactor I would consider would be putting the postgres code and cassandra code in separate files. One way to approach it would be if in the router.js file you check the ENV_DB and require the appropriate file accordingly, if that makes sense. Maybe something like const controller = process.env.ENV_DB === 'pg' ? require('./pg-controllers') : process.env.ENV_DB === 'cass' ? require('./cass-controllers');